### PR TITLE
lock durations passed in constructor instead of hard coded

### PR DESCRIPTION
**Pull request by mejango**

### CODE REVIEW

File: contracts/JBveBanny.sol [hunk 1]
```diff
@@ -78,6 +88,12 @@ contract JBveBanny is ERC721Votes, ERC721Enumerable, Ownable, ReentrancyGuard, J
   */
   uint256 public immutable projectId;
 
+  /** 
+    @notice 
+    The options for lock durations.
+  */
+  uint256[] public lockDurationOptions;
```
Comment (viraj124) [inline contracts/JBveBanny.sol:95]: @mejango should we maybe use an enum instead of an array unless we don't want the duration options to be fixed

**Review by viraj124 — CHANGES_REQUESTED** [review]
Just a suggestion comment mentioned above

File: contracts/JBveBanny.sol (hunk 1 above)
Comment (mejango) [reply contracts/JBveBanny.sol:95]: the idea here is that the duration options are not fixed. They are specifiable by the contract deployer.

Comment (viraj124) [reply contracts/JBveBanny.sol:95]: got it I asked only since earlier it was fixed

**Pull request merged**